2022-02-20 04:31:05,614 INFO    MainThread:24403 [wandb_setup.py:_flush():71] setting env: {}
2022-02-20 04:31:05,614 INFO    MainThread:24403 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-20 04:31:05,614 INFO    MainThread:24403 [wandb_init.py:_log_setup():371] Logging user logs to /home/sohui/code/DTC/code/wandb/run-20220220_043105-2x9l0g5n/logs/debug.log
2022-02-20 04:31:05,614 INFO    MainThread:24403 [wandb_init.py:_log_setup():372] Logging internal logs to /home/sohui/code/DTC/code/wandb/run-20220220_043105-2x9l0g5n/logs/debug-internal.log
2022-02-20 04:31:05,615 INFO    MainThread:24403 [wandb_init.py:init():404] calling init triggers
2022-02-20 04:31:05,615 INFO    MainThread:24403 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'mdoel': 'DTC', 'server': 57, 'image size': '112,112,80'}
2022-02-20 04:31:05,615 INFO    MainThread:24403 [wandb_init.py:init():449] starting backend
2022-02-20 04:31:05,615 INFO    MainThread:24403 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-20 04:31:05,633 INFO    MainThread:24403 [backend.py:ensure_launched():199] starting backend process...
2022-02-20 04:31:05,640 INFO    MainThread:24403 [backend.py:ensure_launched():204] started backend process with pid: 24475
2022-02-20 04:31:05,642 INFO    MainThread:24403 [wandb_init.py:init():458] backend started and connected
2022-02-20 04:31:05,645 INFO    MainThread:24403 [wandb_init.py:init():519] updated telemetry
2022-02-20 04:31:05,647 INFO    MainThread:24403 [wandb_init.py:init():549] communicating current version
2022-02-20 04:31:06,489 INFO    MainThread:24403 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-20 04:31:06,490 INFO    MainThread:24403 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-02-20 04:31:06,752 INFO    MainThread:24403 [wandb_init.py:init():592] starting run threads in backend
2022-02-20 04:31:06,993 INFO    MainThread:24403 [wandb_run.py:_console_start():1816] atexit reg
2022-02-20 04:31:06,994 INFO    MainThread:24403 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2022-02-20 04:31:06,995 INFO    MainThread:24403 [wandb_run.py:_redirect():1695] Redirecting console.
2022-02-20 04:31:07,000 INFO    MainThread:24403 [wandb_run.py:_redirect():1751] Redirects installed.
2022-02-20 04:31:07,000 INFO    MainThread:24403 [wandb_init.py:init():619] run started, returning control to user process
2022-02-20 04:31:14,694 INFO    MainThread:24403 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-02-20 04:31:14,695 INFO    MainThread:24403 [wandb_run.py:_restore():1758] restore
2022-02-20 04:31:17,056 INFO    MainThread:24403 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 590
  total_bytes: 590
}

2022-02-20 04:31:17,259 INFO    MainThread:24403 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 590
  total_bytes: 590
}

2022-02-20 04:31:17,503 INFO    MainThread:24403 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 590
  total_bytes: 590
}

2022-02-20 04:31:17,789 INFO    MainThread:24403 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 590
  total_bytes: 6331
}

2022-02-20 04:31:17,892 INFO    MainThread:24403 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 590
  total_bytes: 6331
}

2022-02-20 04:31:18,003 INFO    MainThread:24403 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 590
  total_bytes: 6331
}

2022-02-20 04:31:18,107 INFO    MainThread:24403 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6304
  total_bytes: 6331
}

2022-02-20 04:31:18,210 INFO    MainThread:24403 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6331
  total_bytes: 6331
}

2022-02-20 04:31:18,314 INFO    MainThread:24403 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6331
  total_bytes: 6331
}

2022-02-20 04:31:18,418 INFO    MainThread:24403 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6331
  total_bytes: 6331
}

2022-02-20 04:31:18,521 INFO    MainThread:24403 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6331
  total_bytes: 6331
}

2022-02-20 04:31:19,033 INFO    MainThread:24403 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6331
  total_bytes: 6331
}

2022-02-20 04:31:19,554 INFO    MainThread:24403 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6331
  total_bytes: 6331
}
local_info {
}

2022-02-20 04:31:20,739 INFO    MainThread:24403 [wandb_run.py:_append_files():2184] logging synced files
